CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+ - Invalid user dictionaries are handled more robustly [[#28][28]
```

Comment:
Looks like an extra square brace in here

```suggestion
 - Invalid user dictionaries are handled more robustly [#28][28]
```

Reply:
I missed the closing bracket. Addressed in 21dd9f76576ef8ab61908826d6dc9639d658e8e5.